docs/Running-behind-nginx.md

```diff
@@ -1,9 +1,43 @@
-# Netdata via nginx
+# Running Netdata Behind Nginx
 
-To pass Netdata via a nginx, use this:
+## Intro
+
+[Nginx](https://nginx.org/en/) is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server used to host websites and applications of all sizes. 
+
+The software is known for its low impact on memory resources, high scalability, and its modular, event-driven architecture which can offer secure, predictable performance.
+
+## Why Nginx
+
+Nginx is used and useful in cases when you do not want to access Netdata via the default `IP ADDRESS:19999`.
+
+With Nginx, the data is accessed via human-readable domains such as `example.com` or `sub.example.com`. 
+
+By default, Nginx is fast and lightweight out of the box.
+
+Nginx can be configured to secure your Netdata insights by the use of authentication to access the dashboard. 
+
+### Nginx configuration file
+
+All Nginx configurations can be found in the `/etc/nginx/` directory. The main configuration file is `/etc/nginx/nginx.conf`. Website or app-specific configurations can be found in the `/etc/nginx/site-available/` directory.
+
+Configuration options in Nginx are known as directives. Directives are organized into groups known as blocks or contexts. The two terms can be used interchangeably.
+
+Depending on your installation source, you’ll find an example configuration file at `/etc/nginx/conf.d/default.conf` or `etc/nginx/sites-enabled/default`, in some cases you may have to manually create the `sites-available` and `sites-enabled` directories. 
+
+You can edit the Nginx configuration file with Nano, Vim or any other text editors you are comfortable with.
+
+After making changes to the configuration files:
+
+- test Nginx configuration with `nginx -t` 
+
+- restart Nginx to effect the change with `/etc/init.d/nginx restart` or `service nginx restart`.
+
+## Ways to pass Netdata via Nginx
 
 ### As a virtual host
 
+With this method instead of `192.0.2.0:19999`, the Netdata dashboard can be accessed via a human-readable URL such as `netdata.example.com` used in the configuration below. 
```

Comment:
What does "instead of SOME RANDOM IP" mean? 
Human-readable URL is not the point here, as explained above.

Reply:
I earlier wrote `123.45.67.89:19999` and @ilyam8 pointed out that I should use IPs reserved for documentation purposes. 
I have replaced it with `IP ADDRESS:19999`. 

The point of the sentence is to tell a user/reader why they would want to access Netdata via Nginx as a virtual host.

Although adding `127.0.1.1	netdata.mydomain.com` to `/etc/hosts` still works fine, we are addressing how it's done with Nginx.

To prevent repetition, I have rewritten the previous sentence in **Why Nginx**.

Reply:
For self-explanatory purposes, I have further changed it from just `IP ADDRESS:19999` to `SERVER_IP_ADDRESS:19999`.